### ISSUE TITLE
Update maven repository

### ISSUE THREAD

**Issue by Janmm14** [WolverinDEV/BungeeUtil#27]
https://oss.sonatype.org/#nexus-search;quick~bungeeutil
Artifacts are from 16 Nov 2016

**Comment by WolverinDEV** [comment]
Yep i know but i have to fit some requirements. Well i'm currently very busy so i don't have realy time to do this.

**Comment by Janmm14** [comment]
ah, okay

**Comment by crysis992** [comment]
Any info when it will be updated?

**Comment by WolverinDEV** [comment]
curretly notink. I dont have time to do this, so i think i wount update the repro.

**Comment by higgs01** [comment]
Any updates on this? would gladly help
